app/app_xdg.go

```diff
@@ -58,13 +58,19 @@ func findFreedestktopColorScheme() fyne.ThemeVariant {
 		return theme.VariantDark
 	}
 
+	// See: https://github.com/flatpak/xdg-desktop-portal/blob/1.16.0/data/org.freedesktop.impl.portal.Settings.xml#L32-L46
+	// 0: No preference
+	// 1: Prefer dark appearance
+	// 2: Prefer light appearance
 	switch value {
```

Comment:
Would it not be better to just keep this to one case? Seems a bit unnecessary with three.
```go
if value == 1 {
    return theme.VariantDark
}

// Default to light theme for 0 and 2 to support Gnome's default see https://github.com/fyne-io/fyne/pull/3561
return theme.VariantLight
```

Reply:
To me it seems good to keep them separate due to the conflicting documentation linked in the method

Reply:
I left the switch to have the code document directly the values that we handle here for `value`
Updated in dc029901192007b3df31bcc09a4670af2806ea69